src/app/shared/components/sidemenu/sidemenu.component.ts

```
@@ -198,12 +198,12 @@ export class SidemenuComponent implements OnInit {
     const { allowedReportsActions, allowedAdvancesActions } = this.allowedActions;
     const teamOptions = [
       {
-        title: 'Team Reports',
+        title: 'Team reports',
         isVisible: allowedReportsActions && allowedReportsActions.approve && showTeamReportsPage,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)